Jarvis-Platform/jarvis-studio
# chore(ItemView): Use ParametersList for update information

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
UI improvements for update information on `runs` & `configurations` ItemView.


## Todos
### General
- [x] Use `ParametersList` instead of `CreateUpdateConfOverview` component

### Conf view
- [x] Add the update information on the right of the tab bar for a Conf View with email + avatar
- [x] Create avatar component (just the initial in a circle like in Gmail if no photo is provided)
- [x] The update date is clickable and show the list of the other update date and updated by from previous configuration version (the previous configuration are available in firestore from the path :
`conf_type/conf_id/history/update_date`)
- [x] When clicking in a previous configuration, replace the current version of the configuration with the previous parameters. Change the color of the update date in red to inform that it is a previous configuration.
- [x] Manage errors in schema for previous configuration which not respect the expected attributs.


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/configuration-view-ui-improvements
```

1. Go to a configuration listing
2. Select a document which has one or many archived configuration
3. See informations
4. Click on timestamp to show archived configuration listing
5. Click on an item => it should update current configuration informations (no save feature for the moment)


## Impacted Areas in Application
- `store`
- `data-workflows`
- `item-mixin`
